src/Products/Annotation/Controllers/AnnotationApiController.cs

```diff
@@ -592,145 +479,38 @@ public HttpResponseMessage Annotate(AnnotationPostedDataEntity annotateDocumentR
             return Request.CreateResponse(HttpStatusCode.OK, annotatedDocument);
         }
 
-        /// <summary>
-        /// Get all annotations from the document
-        /// </summary>
-        /// <param name="documentGuid">string</param>
-        /// <param name="documentType">string</param>
-        /// <returns>AnnotationInfo[]</returns>
-        private AnnotationInfo[] GetAnnotations(string documentGuid, string documentType, string password)
-        {
-            try
-            {
-                FileStream documentStream = new FileStream(documentGuid, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
-                DocumentType docType = DocumentTypesConverter.GetDocumentType(documentType);
-                AnnotationInfo[] annotations = new BaseImporter(documentStream, AnnotationImageHandler, password).ImportAnnotations(docType);
-                documentStream.Dispose();
-                documentStream.Close();
-                return annotations;
-            }
-            catch (System.Exception ex)
-            {
-                throw ex;
-            }
-        }
-
-        private List<PageDataDescriptionEntity> GetAnnotatedPagesForPrint(string path)
-        {
-            AnnotatedDocumentEntity description = new AnnotatedDocumentEntity();
-            try
-            {
-
-                Stream document = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
-                List<PageImage> pageImages = AnnotationImageHandler.GetPages(document, new ImageOptions());
-                document.Dispose();
-                document.Close();
-                for (int i = 0; i < pageImages.Count; i++)
-                {
-                    PageDataDescriptionEntity page = new PageDataDescriptionEntity();
-                    if (pageImages != null)
-                    {
-                        byte[] bytes;
-                        using (MemoryStream memoryStream = new MemoryStream())
-                        {
-                            Stream imageStream = pageImages[i].Stream;
-                            imageStream.Position = 0;
-                            imageStream.CopyTo(memoryStream);
-                            bytes = memoryStream.ToArray();
-                        }
-                        string encodedImage = Convert.ToBase64String(bytes);
-                        page.SetData(encodedImage);
-                    }
-                    description.pages.Add(page);
-                }
-
-                return description.pages;
-            }
-            catch (FileNotFoundException ex)
-            {
-                throw ex;
-            }
-        }
+        public void RemoveAnnotations(string documentGuid, string password) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'RemoveAnnotations' a static method.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)